recipes/thorvg/all/conanfile.py

```diff
@@ -141,6 +152,8 @@ def generate(self):
             tc.project_options["simd"] = bool(self.options.with_simd)
         else:
             tc.project_options["vector"] = bool(self.options.with_vector)
+        if "with_extra" in self.options:
+            tc.project_options["extra"] = str(self.options.with_extra) if self.options.with_extra else ''
```

Comment:
```suggestion
        if self.options.get_safe("with_extra"):
            tc.project_options["extra"] = self.options.with_extra or ''
```

Reply:
I think that we could make it simpler:

```suggestion
        tc.project_options["extra"] = str(self.options.get_safe("with_extra", ""))
```